### DISCUSSION TITLE
Discussions - now enabled!

### DISCUSSION THREAD

**Discussion by martinpaljak** [OpenSC/OpenSC#2774] [General]
# 👋 Welcome to the OpenSC Discussions!

Thanks for being part of the OpenSC community.

This space is for everyone interested in smart cards, PKCS#11, CTK, Windows Minidriver, and secure element technologies. Whether you're a long-time contributor, an integrator, or just getting started, we’re glad you’re here.

## 💬 What you can do here:

- Ask questions about using or building OpenSC
- Share ideas, feature requests, or implementation approaches
- Discuss smart card standards and interoperability
- Showcase projects built with OpenSC
- Connect with other community members

## 🤝 Community Guidelines

Please be respectful, constructive, and collaborative. We aim to foster a welcoming environment where technical discussions can thrive.

If you're reporting a bug or requesting a feature, please use the Issues tab. For everything else — questions, ideas, discussions — you're in the right place!

We look forward to your contributions 🚀

**Comment by Antimatterr** [comment]
hi I was trying to run certutil.exe -scinfo command and 
pkcs11-tool.exe --login --test command but i was getting error message as "No slot" can anyone explain the reson behind it
![image](https://github.com/OpenSC/OpenSC/assets/56473894/6470fa92-a1e1-4b39-97e9-a4745f1e5c21)

**Comment by dengert** [comment]
Looks like Windows PC/SC  is not running. It is used by both cert-util and OpenSC tools. or it could be the reader (or reader in a token) is not supported.
